scripts/src/main/resources/scripts/command/dotnet

```diff
@@ -0,0 +1,63 @@
+﻿#!/usr/bin/env bash
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $1: optional setup
+function doSetup() {
+    
+  dotnetCommand=$(dotnet --version)
+  downloadDotnetUrl="https://dotnet.microsoft.com/download"
+  
+  if [ -z "$dotnetCommand" ]
+  then
+    printf "${RED}dotnet NOT running${NC} \n"
+    printf "${RED}Please install dotnet SDK.${NC} \n"
+    case "$OSTYPE" in
+      solaris*) echo "SOLARIS : not supported platform" ;;
+      darwin*)  open $downloadDotnetUrl ;; 
+      linux*)   exdg-open $downloadDotnetUrl ;; 
+      bsd*)     echo "BSD : not supported platform" ;;
+      msys*)    start $downloadDotnetUrl ;; 
+      *)        echo "unknown: $OSTYPE.  Not supported platform" ;;
```

Comment:
The idea of devonfw-ide is to fully automate the download and installation of the tools and software.
Is that not possible here?
Should we reuse `doIsWindows`, `doIsMacOs` ?
E.g. `doIsWindows` also considers `cygwin`...
You are of course fine to stay with this `case` statement but we need to be aware that we have to keep that in sync with the other stuff and you should then also add `cygwin` to `msys`.

Reply:
I just tried the download for macos and you can simply implement this installation the same way as done for all the other tools (see e.g. vscode, java, etc.).

Reply:
@hohwille many thanks for your comments. I'm going to take a look to your suggestions and update the pull request with the required changes.